source/upgrade-guide/upgrading-elastic-stack/elastic_server_minor_upgrade.rst

```diff
@@ -15,47 +15,26 @@ Prepare the Elastic Stack
       # systemctl stop filebeat
       # systemctl stop kibana
 
-#. Add the new repository for Elastic Stack 7.x:
+#. In case of having disabled the repository for Elastic Stack 7.x it can be enabled using:
```

Comment:
7 or 6?

Reply:
It has to be 7, because it is a minor upgrade from 7.y to 7.x